package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/StandardShaderExporter.cs

```diff
@@ -263,5 +290,12 @@ public class StandardShaderExporter : ShaderExporterBase {
       }
     }
 
+    // local version of the internal BlendMode enum in https://github.com/Unity-Technologies/UnityCsReference/blob/master/Editor/Mono/Inspector/StandardShaderGUI.cs
```

Comment:
Bummer that this is not exposed, but thank you for tracking it down and adding an enum here!